projectsplatform/settings.py

```diff
@@ -21,6 +21,7 @@
 
 
 # SECURITY WARNING: don't run with debug turned on in production!
+SECRET_KEY = 'dulwepf_b#@eh19w)==v+pxvw$u$2gbxpwoevmzg)h-pl1mded'
```

Comment:
@kellim was right, it doesn't run without the secret key.

Reply:
@natsteinmetz Would python-dotenv be acceptable for using in development? https://preslav.me/2019/01/09/dotenv-files-python/

Reply:
@natsteinmetz Let's create a separate story for the secret key and db details.